<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LogDataDialog</class>
 <widget class="QDialog" name="LogDataDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>739</width>
    <height>384</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <widget class="QFrame" name="frame_2">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QPushButton" name="btnLoad">
          <property name="text">
           <string>불러오기</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnSave">
          <property name="text">
           <string>파일 저장</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnClear">
          <property name="text">
           <string>화면 Clear</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="btnDelete">
          <property name="text">
           <string>로그 삭제</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>199</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QFrame" name="frame">
     <property name="minimumSize">
      <size>
       <width>550</width>
       <height>0</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0" colspan="2">
       <widget class="QTextEdit" name="editLog"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="labelCnt">
        <property name="text">
         <string>Log 갯수 :</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="labelMaxCnt">
        <property name="text">
         <string>Log 갯수</string>
        </property>
       </widget>
      </item>
     </layout>
     <zorder>editLog</zorder>
     <zorder>editLog</zorder>
     <zorder>labelMaxCnt</zorder>
     <zorder>labelCnt</zorder>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
